### DISCUSSION TITLE
fan1 speed control

### DISCUSSION THREAD

**Discussion by roei910** [mriscoc/Ender3V2S1#1233] [Q&A]
hello,
i wanted to ask if there is an option to set the fan speed on the fan1 pin, i am currently running a 5015 fan on the hotend.
full speed is very noisy and has some vibrations so i wanted to run it at a lower speed.
was wondering if there is a way to do that with this firmware.

thanks.

**Comment by mriscoc** [comment]
Fan control depends on the board, for example for the Creality 4.2.2 and 4.2.7 the extruder fan is directly connected to power supply, so no control is possible.

**Comment by 3DP-Distributed** [comment]
Is there a way to have my motherboard fan, hotend fan always on but have the part cooling speed still variable? The hotend fan is always on, but it seems the Fan speed control in the firmware controls KFAN-1 & K-FAN2 at the same speed. so basically if I dont want to use part cooling (Nylon, PETG, etc) then the motherboard runs the risk of overheating. I'm using a 4.2.2 board. Thanks!

**Reply by mriscoc** [reply]
Only by rewiring the fan.
